package com.syn.issuetracker.common;

public class ExceptionErrorMessages {

    public static final String VALIDATION_FAILURE = "VALIDATION_FAILURE";
    public static final String DATA_CONFLICT = "Data conflict!";
    public static final String EMAIL_ALREADY_EXISTS = "EMAIL_ALREADY_EXISTS";
    public static final String USERNAME_ALREADY_EXISTS = "USERNAME_ALREADY_EXISTS";
    public static final String TITLE_ALREADY_EXISTS = "TITLE_ALREADY_EXISTS";
    public static final String USER_NOT_FOUND = "User not found!";
    public static final String TASK_NOT_FOUND = "Task not found!";

    public static final String EMAIL_NOT_FOUND = "User with email '%s' was not found!";
}
